### DISCUSSION TITLE
Automatically eliminate bleed-through

### DISCUSSION THREAD

**Discussion by mrichtarsky** [ocrmypdf/OCRmyPDF#1341] [Ideas]
When scanning documents, particularly using ADF, I get noticable bleed-through: I can see the text from the other side of the page shining through.

When scanning double-sided, we have both sides of the page. An automatic process could then, for every scanned page, subtract the corresponding backside of that page (mirrored) to improve the quality of the document.

**Comment by jbarlow83** [comment]
To create this feature I would need some code that reliably detects bleed through, or funding to construct a model for this person, with a lot of data.

**Reply by mrichtarsky** [reply]
Yes, I realize it's not straightforward. Just wanted to gauge interest, originally I submitted the idea to paperless-ngx who told me to try here. I would also like to hack together a prototype, there should be several image procsessing libraries that could be used to implement this. I'm not familiar with OCRmyPDF and the libraries it uses, do you have any recommendations how to start? Thanks!
